build.gradle

```diff
@@ -53,7 +53,7 @@ ext {
 
 allprojects {
 
-    group = 'org.opensearch'
+    group = 'org.opensearch.plugin'
```

Comment:
This is not ideal because we set `group=org.opensearch` so jars can be pushed to `org/opensearch/<>`.
We then set `groupId=org.opensearch.plugin` so zips can be pushed to `org/opensearch/plugin/<>`.

Reply:
For k-NN, we dont puish any jars at the moment.

Reply:
@jmazanec15 you could keep `group=org.opensearch` by there and use `groupId=org.opensearch.plugin` for ZIPs, it will fix the issue:
```
   > Invalid publication 'nebula': supplied groupId (org.opensearch) does not match value from POM file (org.opensearch.plugin). Cannot edit groupId directly in the POM file.
```

Reply:
@reta I believe this is what we do now:
1. https://github.com/opensearch-project/k-NN/blob/main/build.gradle#L56
2. https://github.com/opensearch-project/k-NN/blob/main/build.gradle#L91

I think I can leave as is, but change build script to use: